src/compiler/crystal/config.cr

```diff
@@ -14,11 +14,13 @@ module Crystal
 
     def self.description
       version, sha = version_and_sha
-      if sha
```

Comment:
Version string is already a method. This is explicitly only used to print `crystal -v` and isn't exposed anywhere else.